docs/supported_platforms_and_configurations.md

```diff
@@ -65,7 +65,7 @@ For more information see [conan-io/conan-docker-tools](https://github.com/conan-
 
 ## MacOS
 
-- Compilers: Apple-clang versions 11.0.0, 12.0.0, 12.0.0
+- Compilers: Apple-clang versions 11.0.0, 12.0.0, 12.0.0, 13.0.0
```

Comment:
```suggestion
- Compilers: Apple-clang versions 11.0.0, 12.0.0, 13.0.0
```

Reply:
Indeed, but the bot will insist with a new PR.